composer.json

```diff
@@ -14,13 +14,24 @@
     }
   ],
   "autoload": {
-    "psr-0": {
-      "PAMI": "src/mg"
+    "psr-4": {
+      "PAMI\\": "src/mg/PAMI"
     }
   },
   "require": {
     "php": ">=5.3.3",
     "apache/log4php": "2.3.0",
     "psr/log": "~1.0"
+  },
+  "require-dev": {
+    "phpunit/phpunit": "4.8.*",
```

Comment:
The official recommendation of the composer project is to use the `^<major.minor>` syntax. So in this case that would be `"phpunit/phpunit": "^4.8"`. That works for all packages implementing semantic versioning.